usage/app/lib/Config.scala

```diff
@@ -85,7 +85,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
       .withRegion(Regions.EU_WEST_1)
 
   val postfix = if (stage == "DEV") {
-    iamClient.getUser().getUser().getUserName()
+    "b0rked"
```

Comment:
the reason i included it was that iamClient.getUser().getUser().getUserName() was causing an AWS permissions error when loading.  I agree that we could possibly change the string "b0rked" to something a little less irreverant, but I think we still need a static string value here unfortunately